packages/dom-ready/src/index.js

```diff
@@ -15,6 +15,9 @@
  * @return {void}
  */
 const domReady = function( callback ) {
+	if ( typeof document === 'undefined' ) {
+		return;
```

Comment:
should we call the callback instead? warn? assume folks using this are using it in an environment that supports the dom?

Reply:
> should we call the callback instead? warn? assume folks using this are using it in an environment that supports the dom?

Hm. I think the callback should be called, yes. I don't feel strongly about it, but I think a warning message would be reasonable as well to inform the developer that a document was not found.

Reply:
> Hm. I think the callback should be called, yes. I don't feel strongly about it, but I think a warning message would be reasonable as well to inform the developer that a document was not found.

We should implement this.